src/config/constants/farmAuctions.ts

```diff
@@ -1,130 +1,182 @@
+import { Token, Pair, ChainId } from '@pancakeswap/sdk'
+import { WBNB, BUSD } from './tokens'
 import { FarmAuctionBidderConfig } from './types'
 
+const getLpAddress = (token0: string, token1: Token) => {
+  const token0asToken = new Token(ChainId.MAINNET, token0, 18)
+  return Pair.getAddress(token0asToken, token1)
+}
+
 export const whitelistedBidders: FarmAuctionBidderConfig[] = [
   {
     account: '0x9Ed5a62535A5Dd2dB2d9bB21bAc42035Af47F630',
     farmName: 'NAV-BNB',
+    tokenAddress: '0xbfef6ccfc830d3baca4f6766a0d4aaa242ca9f3d',
+    quoteToken: WBNB,
```

Comment:
Think it would make more sense to re-use the same `tokens` as the farms config use (since the project is `farm-auctions`); so `tokens.wbnb` and `tokens.busd`

Reply:
I think this is a good idea.